createmap/src/App.css

```diff
@@ -299,3 +299,5 @@ body,
 .gridless .grid-row {
 	border: none;
```

Comment:
whats this?

Reply:
It is a class to remove the grid borders in the create map grid layout.